.github/workflows/ci.yml

```diff
@@ -160,38 +160,6 @@ jobs:
         run: |
           time make test262
 
-  linux-gcc48:
```

Comment:
Can we not save this one? If you are worried about the CMake checks, we could remove 262 from all and have it compiled when we run the test262* targets, WDYT?

Reply:
Added a version check to CMakeLists.txt, hope that's good enough.